spec.emu

```diff
@@ -326,6 +597,57 @@ contributors: Luca Casonato, Guy Bedford
           </table>
         </emu-table>
 
+
+        <emu-clause id="sec-parsemodule" type="abstract operation">
+          <h1>
+            ParseModule (
+              _sourceText_: ECMAScript source text,
+              _realm_: a Realm Record,
+              _hostDefined_: anything,
+            ): a Source Text Module Record or a non-empty List of *SyntaxError* objects
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It creates a Source Text Module Record based upon the result of parsing _sourceText_ as a |Module|.</dd>
+          </dl>
+          <emu-alg>
+            1. Let _body_ be ParseText(_sourceText_, |Module|).
+            1. If _body_ is a List of errors, return _body_.
+            1. Let _requestedModules_ be the ModuleRequests of _body_.
+            1. <ins>Let _recursiveRequestedModules_ be the ModuleRequests of _body_ with argument *true*</ins>.
+            1. <ins>Let _requestedModuleReflections_ be a list containing the elements of _requestedModules_ that are not elements of _recursiveRequestedModules_
```

Comment:
To be clearer perhaps let's call this `_requestedModuleReflectionsOnly_` or similarly update the naming / documentation for this field since it is not a list of all of the module reflections now.

Reply:
Done, I also added an `<emu-note>` with an example.